test/waittask.spec.js

```diff
@@ -189,7 +182,8 @@ module.exports.addTests = function({testRunner, expect, product, puppeteer}) {
       await page.evaluate(() => window.__injected = true);
       await watchdog;
     });
-    it('should survive cross-process navigation', async({page, server}) => {
+    // https://chromium-review.googlesource.com/c/chromium/src/+/1766001/21#message-a6a61261b97ffca6ecb180c0a2303b538f7a6476
```

Comment:
I didn't look too closely, but I'd guess you need to catch this new error string in the navigation watcher.

Reply:
Thanks for the hint. I think you meant `ExecutionContext`, and I'll add the check there. Let me know if I misunderstood!